noir-projects/noir-protocol-circuits/crates/blob/src/utils/compress_to_blob_commitment.nr

```diff
@@ -90,6 +90,25 @@ mod tests {
         assert_eq(reconstructed_y, point.y);
     }
 
+    #[test]
+    unconstrained fn infinity_point_compression() {
+        let point = BLSPoint::point_at_infinity();
+        let (flags, x) = get_flags(point);
+        // is_compressed = true
+        assert_eq(flags[0], true);
+        // is_infinity = true
+        assert_eq(flags[1], true);
+        // is_greater = false (is infinity)
+        assert_eq(flags[2], false);
```

Comment:
Interestingly, this flag could accidentally be `true` if the point at infinity were to be expressed as `{x: 0, y: 1, is_infinity: true}`, since `is_greater = y > -y`. So that's another good reason for ensuring the canonical form of infinity.